src/utils/user_utils.js

```diff
@@ -21,21 +21,21 @@ export function getFullName(user: UserProfile): string {
 export function displayUsername(
     user: UserProfile,
     teammateNameDisplay: string,
-    usernameWithPrefix: boolean,
+    useDefaultUserName: boolean = true,
```

Comment:
this name is misleading, cause I'm seeing this returning `Someone` instead of the username, also why are we removing the `usernameWithPrefix` ?

Reply:
i  can change this to `fallbackUsername` if that makes sense to you. If you have anything in mind please do let me know

Reply:
that works for me, or something like `allowAnonymous`